Sources/ComposableArchitecture/Documentation.docc/Tutorials/BuildingSyncUps/06-SyncUpDetail/TestingSyncUpDetail.tutorial

```diff
@@ -70,4 +70,69 @@
       we did in <doc:PresentingSyncUpForm>, but we will leave that as an exercise for the reader.
     }
   }  
+
```

Comment:
📓 This is the "delete flow" tutorial section back with the shared stuff. Hope you'll like the way I described it. If not feel free to update it of completely remove it again until you'll have time for a better one. Both are ok for me!